src/ooxml/java/org/apache/poi/xssf/streaming/TempFileSharedStringsTable.java

```diff
@@ -54,6 +54,7 @@ Licensed to the Apache Software Foundation (ASF) under one or more
  */
 class TempFileSharedStringsTable extends SharedStringsTable {
 
+    private static final Random RND = new Random();
```

Comment:
Since this is used to generate an encryption key, shouldn't this be SecureRandom?

Reply:
Reasonable point. I've updated this.